R/param_class_weights.R

```diff
@@ -0,0 +1,22 @@
+#' Parameters for class weights for imbalanced problems
+#'
+#' This parameter can be used to moderate how much influence certain classes
+#' receive during training.
+#'
+#' @inheritParams Laplace
+#' @details
+#' Used in `lantern::lanter_logistic_reg()` and `lantern::lanter_mlp()`
+#' @examples
+#' class_weights()
+#' @export
+class_weights <- function(range = c(1, 10), trans = NULL) {
```

Comment:
do we want a range of [0,1]?

Reply:
There's really no upper bound; I capped it at 10

Reply:
ah, I guess I'm just so used to weights being [0,1] but if that's intentional here, that's fine! :+1: